blog/en/blog/2024/04/29/release-apache-apisix-3.8.1.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Release Apache APISIX 3.8.1"
+authors:
+  - name: "Yuansheng Wang"
```

Comment:
Hi @AlinsRan - should this be you or? Who's working on the release this time

Reply:
I'm just helping.